deployment-templates/deploy-forseti.yaml.in

```diff
@@ -73,21 +75,14 @@ resources:
     # e.g. "I want to deploy release 1.0"
     #    # branch-name: "master" <-- COMMENT OUT
     #    release-version: "1.0"
-    {BRANCH_OR_RELEASE}
+    branch-name: BRANCH_NAME
```

Comment:
not sure why this was put back. This file is meant to be used as an input file for the prereq wizard